janusgraph-dist/src/assembly/descriptor/common-full.component.xml

```diff
@@ -64,15 +64,5 @@
                 <exclude>${project.groupId}:janusgraph-core:jar:javadoc:${project.version}</exclude>
             </excludes>
         </dependencySet>
-
-        <!-- Example data (from janusgraph-hadoop/faunus) -->
-        <dependencySet>
```

Comment:
Could you briefly explain what the purpose of this was? Was it part of the packaging?

Reply:
I have no idea.

Reply:
But you are sure that we don't need it any more?

Reply:
The folder doesn't exists.

Reply:
I will check something.

Reply:
Looks good.